website/util/sanitize.py

```diff
@@ -11,6 +11,10 @@ def strip_html(unclean):
     :return: stripped string
     :rtype: str
     """
+    # Handle non string inputs so this function can be used with higher-order
+    # functions, such as rapply (recursively applies a function to collections)
+    if not isinstance(unclean, basestring):
+        return unclean
```

Comment:
Yes, I agree that collection types should be bleached. However, bools, numbers, and date/time value should all be noop. So how about 

```python
# Still allow `None` to be bleached, since some code depends on this returning a string
# e.g. strip_html(value).lower()
if not is_iterable_but_not_string(unclean) and unclean is not None:
    return unclean
```